Updates and clean up for 1.1.0 release documentation (#7356)
magento/devdocs: src/cloud/docker/docker-split-db.md

---
group: cloud-guide
title: Enable Magento split database solution
functional_areas:
- Cloud
- Setup
- Configuration
- Split database performance solution
---

The Magento split database performance solution improves the scalability of Magento by providing three separate databases to manage different functional areas of the Magento application.  The following instructions show how to configure this solution in the {{site.data.var.mcd-prod}} environment, exporting the Magento Sales and Magento Quote data from the main database and creating two separate databases in the Cloud Docker environment.

{:.bs-callout-info}

> See the [Magento split database performance solution][] guide for detailed information and extended examples.

Reviewer: Need to inform them it is in a different guide.

```suggestion
See the [Magento split database performance solution][] topic in the _Configuration_ guide for detailed information and extended examples.
```

{:.procedure}

> Add split DB configuration to {{site.data.var.ece}} project configuration

Reviewer: Semantics aside, configuration was used twice.

```suggestion
To add split DB to {{site.data.var.ece}} project configuration:
```

   ...
   ```

> 1. In the magento.app.yaml` file, add the relationships for the additional databases.

Reviewer: ```suggestion
1. In the `.magento.app.yaml` file, add the relationships for the additional databases.
```

   ...
   ```

> 1. Verify that the 'MAGENTO_CLOUD_RELATIONSHIPS' variable returns the database access credentials.

Reviewer: Change from apostrophes

```suggestion
1. Verify that the `MAGENTO_CLOUD_RELATIONSHIPS` variable returns the database access credentials.
```
